src/geowombat/core/sops.py

```diff
@@ -1157,65 +1273,93 @@ def coregister(
                 raise KeyError
 
         with tempfile.TemporaryDirectory() as tmp:
-            ref_path = Path(tmp) / '_reference.tif'
-            tar_path = Path(tmp) / '_target.tif'
-            with gw_.open(reference.filename, chunks={'band': -1, 'y': 256, 'x': 256}) as ref_src:
+            ref_path = Path(tmp) / "_reference.tif"
+            tar_path = Path(tmp) / "_target.tif"
+            ref_kwargs = (

```

Comment:
Use `h5netcdf` engine if the file is NetCDF.

Reply:
My environment throws many errors if the this NetCDF engine is not used.

Reply:
yeah netcdfs are amazing but fickle